pkg/app/app.go

```diff
@@ -61,12 +61,13 @@ func initConfiguration(engine workflow.Engine, config configuration.Configuratio
 		client := engine.GetNetworkAccess().GetHttpClient()
 		url := config.GetString(configuration.API_URL)
 		apiClient.Init(url, client)
-		if existingValue != nil && len(existingValue.(string)) > 0 {
-			orgId := existingValue.(string)
+		existingString, ok := existingValue.(string)
```

Comment:
Question: is it safe in golang to do a type conversion without checking if the value is nil?

Reply:
Yes. https://go.dev/play/p/Lm_pTMxJLRl